Sources/Auth/Core/SocialAuthenticationManager.swift

```diff
@@ -27,8 +35,8 @@ extension SocialAuthenticationManager {
     authenticators.forEach { $0.signOut() }
   }
   
-  public var isAuthenticated: PovioKitPromise.Promise<Authenticator.Authenticated> {
-    all(promises: authenticators.map { $0.isAuthenticated })
-      .map { $0.contains(true) }
+  public func canOpenUrl(_ url: URL, application: UIApplication, options: [UIApplication.OpenURLOptionsKey : Any]) -> Bool {
+    guard let activeAuthenticator = authenticators.first(where: { $0.isAuthenticated }) else { return false }
```

Comment:
You can use the computed property instead.

Reply:
Indeed 👍